config/secrets.yml

```diff
@@ -1,7 +1,5 @@
 default: &default
   secret_key_base: MGhvYnpIWnJaRHhLaDJrRkpOd1JqWFhHUHMuRFpVeGRhRE5GVHY1NDdBQkwxM0NkWGdnY1NnZmhNMWhCRzBBRG80U2YuTG5hMURrb3NkZ2VURkJOelZkZmhFMUM0TWJwdGFwcFJhM21hRHdQVTFoRUpWZXZCbmp3YnROZkZ0a2MK
-  http_auth_username: <%= ENV["SP_NAME"] %>
```

Comment:
ah ok, so these are being used to basic auth to the IDP endpoints? 

Perhaps we can save them as regular env vars / Figaro env vars rather than putting them in `secrets.yml` ?